The interface appears in the configuration

### Describe the bug

The leastActiveRequestLoadBalancer uses P2C to select the host, and supports the number of random selections configured through xds. But if it is configured through the configuration file, the following error will occur

```
=== RUN   TestClusterUnmarshal
    unmarshal_test.go:114: json: cannot unmarshal object into Go struct field Cluster.lbconfig of type v2.IsCluster_LbConfig
--- FAIL: TestClusterUnmarshal (0.00s)
```

The reason is that the interface type appears in the configuration type, which makes it impossible to unmarshal correctly

https://github.com/mosn/mosn/blob/35258919244aa5c751d036af199414824230c230/pkg/config/v2/upstream.go#L107

https://github.com/mosn/mosn/blob/35258919244aa5c751d036af199414824230c230/pkg/config/v2/loadbalancer.go#L31-L33

### Expected behavior

Users can configure the number of P2C random selections through the configuration file

### Actual behavior

Error occured when using configuration file

### Steps to reproduce

```json
{
  "cluster": {
    "lbconfig": {
      "choice_count": 10
    }
  }
}
```

### Minimal yet complete reproducer code (or GitHub URL to code)

It seems no need to using interface to support polymorphism, just using a struct named `LbConfig` to included all fields.

```go
	LbConfig             *LbConfig  `json:"lbconfig,omitempty"`
```

```go
type LbConfig struct {
	ChoiceCount uint32 `json:"choice_count"`
}
```

### Environment

- MOSN Version

--- a/istio/_istio152/xds/conv/convertxds.go
+++ b/istio/_istio152/xds/conv/convertxds.go
@@ -226,10 +226,10 @@ func convertDnsLookupFamily(family xdsapi.Cluster_DnsLookupFamily) v2.DnsLookupF
 }
 
 // TODO support more LB converter
-func convertLbConfig(config interface{}) v2.IsCluster_LbConfig {
+func convertLbConfig(config interface{}) *v2.LbConfig {
 	switch config.(type) {
 	case *xdsv2.Cluster_LeastRequestLbConfig:
-		return &v2.LeastRequestLbConfig{ChoiceCount: config.(*xdsv2.Cluster_LeastRequestLbConfig).ChoiceCount.GetValue()}
+		return &v2.LbConfig{ChoiceCount: config.(*xdsv2.Cluster_LeastRequestLbConfig).ChoiceCount.GetValue()}
 	default:
 		return nil
 	}

--- a/istio/istio1106/xds/conv/convert_cluster.go
+++ b/istio/istio1106/xds/conv/convert_cluster.go
@@ -92,10 +92,10 @@ func ConvertClustersConfig(xdsClusters []*envoy_config_cluster_v3.Cluster) []*v2
 }
 
 // TODO support more LB converter
-func convertLbConfig(config interface{}) v2.IsCluster_LbConfig {
+func convertLbConfig(config interface{}) *v2.LbConfig {
 	switch config.(type) {
 	case *envoy_config_cluster_v3.Cluster_LeastRequestLbConfig:
-		return &v2.LeastRequestLbConfig{ChoiceCount: config.(*envoy_config_cluster_v3.Cluster_LeastRequestLbConfig).ChoiceCount.GetValue()}
+		return &v2.LbConfig{ChoiceCount: config.(*envoy_config_cluster_v3.Cluster_LeastRequestLbConfig).ChoiceCount.GetValue()}
 	default:
 		return nil
 	}

--- a/pkg/config/v2/loadbalancer.go
+++ b/pkg/config/v2/loadbalancer.go
@@ -21,15 +21,8 @@ import (
 	"mosn.io/api"
 )
 
-type LeastRequestLbConfig struct {
-	ChoiceCount uint32
-}
-
-func (lbconfig *LeastRequestLbConfig) isCluster_LbConfig() {
-}
-
-type IsCluster_LbConfig interface {
-	isCluster_LbConfig()
+type LbConfig struct {
+	ChoiceCount uint32 `json:"choice_count,omitempty"`
 }
 
 type HashPolicy struct {

--- a/pkg/config/v2/unmarshal_test.go
+++ b/pkg/config/v2/unmarshal_test.go
@@ -103,7 +103,10 @@ func TestClusterUnmarshal(t *testing.T) {
 					}
 				}
 			}
-		]
+		],
+		"lbconfig": {
+			"choice_count": 10
+  		}
 	}`
 	b := []byte(clusterConfig)
 	cluster := &Cluster{}

--- a/pkg/config/v2/upstream.go
+++ b/pkg/config/v2/upstream.go
@@ -104,7 +104,7 @@ type Cluster struct {
 	Hosts                []Host              `json:"hosts,omitempty"`
 	ConnectTimeout       *api.DurationConfig `json:"connect_timeout,omitempty"`
 	IdleTimeout          *api.DurationConfig `json:"idle_timeout,omitempty"`
-	LbConfig             IsCluster_LbConfig  `json:"lbconfig,omitempty"`
+	LbConfig             *LbConfig           `json:"lbconfig,omitempty"`
 	DnsRefreshRate       *api.DurationConfig `json:"dns_refresh_rate,omitempty"`
 	RespectDnsTTL        bool                `json:"respect_dns_ttl,omitempty"`
 	DnsLookupFamily      DnsLookupFamily     `json:"dns_lookup_family,omitempty"`

--- a/pkg/mock/upstream.go
+++ b/pkg/mock/upstream.go
@@ -906,10 +906,10 @@ func (mr *MockClusterInfoMockRecorder) IdleTimeout() *gomock.Call {
 }
 
 // LbConfig mocks base method.
-func (m *MockClusterInfo) LbConfig() v2.IsCluster_LbConfig {
+func (m *MockClusterInfo) LbConfig() *v2.LbConfig {
 	m.ctrl.T.Helper()
 	ret := m.ctrl.Call(m, "LbConfig")
-	ret0, _ := ret[0].(v2.IsCluster_LbConfig)
+	ret0, _ := ret[0].(*v2.LbConfig)
 	return ret0
 }
 

--- a/pkg/types/upstream.go
+++ b/pkg/types/upstream.go
@@ -221,7 +221,7 @@ type ClusterInfo interface {
 	LbOriDstInfo() LBOriDstInfo
 
 	// Optional configuration for the load balancing algorithm selected by
-	LbConfig() v2.IsCluster_LbConfig
+	LbConfig() *v2.LbConfig
 
 	//  Optional configuration for some cluster description
 	SubType() string

--- a/pkg/upstream/cluster/cluster.go
+++ b/pkg/upstream/cluster/cluster.go
@@ -219,7 +219,7 @@ type clusterInfo struct {
 	tlsMng               types.TLSClientContextManager
 	connectTimeout       time.Duration
 	idleTimeout          time.Duration
-	lbConfig             v2.IsCluster_LbConfig
+	lbConfig             *v2.LbConfig
 	slowStart            types.SlowStart
 }
 
@@ -278,7 +278,7 @@ func (ci *clusterInfo) LbOriDstInfo() types.LBOriDstInfo {
 	return ci.lbOriDstInfo
 }
 
-func (ci *clusterInfo) LbConfig() v2.IsCluster_LbConfig {
+func (ci *clusterInfo) LbConfig() *v2.LbConfig {
 	return ci.lbConfig
 }
 

--- a/pkg/upstream/cluster/lb_leastconnection.go
+++ b/pkg/upstream/cluster/lb_leastconnection.go
@@ -18,7 +18,6 @@
 package cluster
 
 import (
-	v2 "mosn.io/mosn/pkg/config/v2"
 	"mosn.io/mosn/pkg/types"
 )
 
@@ -31,7 +30,7 @@ type leastActiveConnectionLoadBalancer struct {
 func newleastActiveConnectionLoadBalancer(info types.ClusterInfo, hosts types.HostSet) types.LoadBalancer {
 	lb := &leastActiveConnectionLoadBalancer{}
 	if info != nil && info.LbConfig() != nil {
-		lb.choice = info.LbConfig().(*v2.LeastRequestLbConfig).ChoiceCount
+		lb.choice = info.LbConfig().ChoiceCount
 	} else {
 		lb.choice = default_choice
 	}

--- a/pkg/upstream/cluster/loadbalancer.go
+++ b/pkg/upstream/cluster/loadbalancer.go
@@ -28,7 +28,6 @@ import (
 	"github.com/trainyao/go-maglev"
 
 	"mosn.io/api"
-	v2 "mosn.io/mosn/pkg/config/v2"
 	"mosn.io/mosn/pkg/log"
 	"mosn.io/mosn/pkg/types"
 	"mosn.io/pkg/variable"
@@ -247,7 +246,7 @@ type leastActiveRequestLoadBalancer struct {
 func newleastActiveRequestLoadBalancer(info types.ClusterInfo, hosts types.HostSet) types.LoadBalancer {
 	lb := &leastActiveRequestLoadBalancer{}
 	if info != nil && info.LbConfig() != nil {
-		lb.choice = info.LbConfig().(*v2.LeastRequestLbConfig).ChoiceCount
+		lb.choice = info.LbConfig().ChoiceCount
 	} else {
 		lb.choice = default_choice
 	}

